prow/github/client.go

```diff
@@ -2686,13 +2701,18 @@ func (c *Client) CreateProjectCard(columnID int, projectCard ProjectCard) (*Proj
 	if c.dry {
 		return &projectCard, nil
 	}
+	reqParams := struct {
+		ContentID   int    `json:"content_id"`
+		ContentType string `json:"content_type"`
+	}{projectCard.ContentID, projectCard.ContentType}
+
 	path := fmt.Sprintf("/projects/columns/%d/cards", columnID)
 	var retProjectCard ProjectCard
 	_, err := c.request(&request{
 		method:      http.MethodPost,
 		path:        path,
 		accept:      "application/vnd.github.inertia-preview+json",
-		requestBody: &projectCard,
+		requestBody: &reqParams,
```

Comment:
Why does `&projectCard` not work here and we have to create a new one? Doesn't `reqParams` have the same fields as `projectCard `?

Reply:
true, fixed. Now that ProjectCard has both these relevant fields code is changed as suggested. Thanks for the review.